src/generate_stochastic_block_model_with_features.py

```diff
@@ -25,7 +25,7 @@
 
 import numpy as np
 
-from graph_world.nodeclassification.sbm_simulator import GenerateStochasticBlockModelWithFeatures
```

Comment:
I believe this file is here by mistake, and is not used or called by any other module. If you agree, please revert this change, and I will remove the file later. If not, let me know what this is doing!